staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -397,10 +395,6 @@ type fancyResponseWriterDelegator struct {
 	*ResponseWriterDelegator
 }
 
-func (f *fancyResponseWriterDelegator) CloseNotify() <-chan bool {
```

Comment:
we still have vendored dependencies that use this, and dropping the implementation from our writer wrapper will break those. I'm hesitant to drop this until there are no uses within vendor